sdk-tests/src/test/java/io/dapr/it/actors/app/MyActor.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.it.actors.app;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public interface MyActor {
```

Comment:
Can you give it a name that is specific to your test? Also, we are having those in services/springboot.